core/src/main/java/org/openedx/core/data/api/CourseApi.kt

```diff
@@ -6,19 +6,20 @@ import retrofit2.http.*
 
 interface CourseApi {
 
-    @GET("/mobile_api_extensions/v1/users/{username}/course_enrollments")
+    @GET("/api/mobile/v3/users/{username}/course_enrollments/")
```

Comment:
I believe it's a great suggestion 👍👍👍
It might be beneficial to handle this improvement in a separate pull request to include a user API version to the config as well.
What do you think?

Reply:
sure,
@moiz994 can you please create a ticket for these improvements ?

Reply:
No need to create the ticket for now, we will handle it later